code/datums/quirks/neutral.dm

```diff
@@ -316,7 +316,10 @@
 	// Only tongues of people with this quirk can't be removed. Manually spawned or found tongues can be.
 	new_tongue.organ_flags |= ORGAN_UNREMOVABLE
 
-	give_item_to_holder(/obj/item/clothing/gloves/radio, list(LOCATION_GLOVES = ITEM_SLOT_GLOVES, LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
+	if(isplasmaman(human_holder))
+		give_item_to_holder(/obj/item/clothing/gloves/color/plasmaman/radio, list(LOCATION_GLOVES = ITEM_SLOT_GLOVES, LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
+	else
+		give_item_to_holder(/obj/item/clothing/gloves/radio, list(LOCATION_GLOVES = ITEM_SLOT_GLOVES, LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
```

Comment:
since the `give_item_to_holder()` proc is pretty much copy+pasted, you could probably generalize it

something like
```
var/glove_path = /obj/item/clothing/gloves/radio
if(isplasmaman())
  glove_path = /obj/item/clothing/gloves/color/plasmaman/radio

give_item_to_holder(glove_path, ...)
```

Not necessary, just an idea.

Reply:
no you're right that's a better idea, give me a sec

Reply:
it worked, thanks